open-api/src/testFixtures/java/org/apache/iceberg/rest/RCKUtils.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.rest;
+
+import org.apache.iceberg.relocated.com.google.common.collect.Maps;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.iceberg.CatalogProperties;
+import org.eclipse.jetty.server.Server;
+
+class RCKUtils {
+  private static final String CATALOG_ENV_PREFIX = "CATALOG_";
+  static final String RCK_REQUIRES_NAMESPACE_CREATE = "rck.requires-namespace-create";
+  static final String RCK_SUPPORTS_SERVERSIDE_RETRY = "rck.supports-serverside-retry";
+
+  private static Server localServer;
+
+  private RCKUtils() {}
+
+  /**
+   * Utility method that allows configuring catalog properties via environment variables.
```

Comment:
It might be nice to add a section to `open-api/README.md` with how to use the TCK. This makes it easier to find/read than from the Java classes itself. WDYT?

Reply:
Great idea.  I added a section with examples of how to run in the README.